scene/3d/navigation_agent_3d.cpp

```diff
@@ -192,15 +192,15 @@ Vector3 NavigationAgent3D::get_target_location() const {
 Vector3 NavigationAgent3D::get_next_location() {
 	update_navigation();
 	if (navigation_path.size() == 0) {
-		ERR_FAIL_COND_V(agent_parent == nullptr, Vector3());
+		ERR_FAIL_COND_MSG(agent_parent == nullptr,  "The agent has no parent");
 		return agent_parent->get_global_transform().origin;
 	} else {
 		return navigation_path[nav_path_index] - Vector3(0, navigation_height_offset, 0);
 	}
 }
 
 real_t NavigationAgent3D::distance_to_target() const {
-	ERR_FAIL_COND_V(agent_parent == nullptr, 0.0);
+	ERR_FAIL_COND_MSG(agent_parent == nullptr,  "The agent has no parent");
```

Comment:
Add a period at the end for consistency:

```suggestion
	ERR_FAIL_COND_MSG(agent_parent == nullptr,  "The navigation agent has no parent.");
```

Reply:
Updated